arch/arm64/src/imx9/hardware/imx93/imx93_memorymap.h

```diff
@@ -56,6 +56,7 @@
 #define IMX9_FLEXIO1_BASE                  (0x425C0000UL)
 #define IMX9_FLEXIO2_BASE                  (0x425D0000UL)
 #define IMX9_FLEXSPI_BASE                  (0x425E0000UL)
+#define IMX9_FLEXSPI_PERIPHERAL_BASE       (0x28000000UL)
```

Comment:
I don't think this is needed if chip.h already defines CONFIG_FSPI_PER_BASEADDR ?

Reply:
fixed